src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -2090,12 +2090,6 @@ namespace OpenMS
           {
             temp_aa_sequence.setNTerminalModification(mod.getRegisteredMod());
           }
```

Comment:
why removed?

Reply:
Happens all the time if you have well written pepXML. It is just a fall-back in case the modification was not annotated at the peptide level.